dmcontent/questions.py

```diff
@@ -776,6 +788,28 @@ def value(self):
 
         return value
 
+    @property
+    def filter_value(self):
+        # Display values for options where we want to show different text ('filter_label') than the ususal 'label'
+        options = self.get('options')
+        value = self._service_data.get(self.id, '')
+        if options and value:
+            value = list(value)  # Make a copy of the list to avoid mutating the underlying list data
+            for i, v in enumerate(value):
+                for option in options:
+                    if 'filter_label' in option and 'value' in option and v == option['value']:
+                        value[i] = option['filter_label']
```

Comment:
Can we just do `value[i] = option['filter_label'] or option['label']` and then we only need one branch?